### DISCUSSION TITLE
Hook client folder vue pug changes to server production start

### DISCUSSION THREAD

**Discussion by a-nai** [requarks/wiki#2819] [Help / Questions] [answered]
I try to change page vue pug
wiki/client/themes/default/components/nav-sidebar.vue

          v-list-item-title {{ item.title }}

this line
And after do 

sudo node server

and nothing happens - i mean client folder totally not connected to dev action in real site.

**Comment by NGPixel** [comment, marked as answer]
Any change to the client files need to be recompiled:

https://docs.requarks.io/dev#building-production-assets
